oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -491,6 +482,12 @@ public boolean createScopedRequired() {
     return scopes.isEmpty() && defaultScopes.isEmpty();
   }
 
+  /** Returns true if credential is configured domain wide delegation */
+  @VisibleForTesting
+  boolean isConfiguredForDomainWideDelegation() {
```

Comment:
nit: could this just be named hasServiceAccountUser or something like that instead? Makes sense to me to have individual functions that just do what they say, and then if we end up with more checks later for Domain wide delegation we could have a method that combines the smaller methods.

Reply:
Hopefully it makes sense )